_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+* [US-RSE Financials](#financials)
+* [2022 Steering Committee Roles](#steering-committee)
+* [Get Involved](#involved)
+
+<a name="survey"></a>
+# **International RSE Survey**
+
+The [2021 RSE Survey](https://softwaresaved.limequery.com/386272?lang=en) is still open. You don’t need to be called an RSE to complete the survey. Your formal job title does not matter: we want to hear from you if you develop software for academic research as part of your work. The survey provides considerable detail about the RSE role: a person’s route into the role, the type of work they conduct, their working practices, job satisfaction and demographics. This means it takes around 20 minutes to complete. We appreciate that this is a significant amount of your time to donate - and we thank you for it!
+
+We strongly encourage you to respond to the survey and to share it with others who may be interested. The results of this survey will be greatly beneficial to US-RSE as we move forward. The more responses we have, the better our knowledge.
+
+For more details on the survey visit [https://software.ac.uk/news/complete-international-rse-survey-and-help-us-understand-rse-community](https://software.ac.uk/news/complete-international-rse-survey-and-help-us-understand-rse-community).
+
+You can complete the 2021-2022 RSE survey in [English](https://softwaresaved.limequery.com/386272?lang=en), [French](https://softwaresaved.limequery.com/386272?lang=fr), [German](https://softwaresaved.limequery.com/386272?lang=de-informal), or [Spanish](https://softwaresaved.limequery.com/386272?lang=es).
+
+<a name="workshop"></a>
+# **US-RSE Community Workshop**
+
+We are excited to announce that the first US-RSE Association [community building workshop](https://us-rse.org/first-community-workshop/) will be held April 26-27, 2022 in Princeton, NJ, after two years of delays due to COVID-19! This will be a workshop focused on planning the best path forward to grow the US-RSE Association.
+
+<a name="community-calls"></a>
+# **US-RSE Community Calls**
+
+The February community call will be on February 10, 2022 at 12 ET. We will be talking about Testing Frameworks and Processes.
+
+Most developers will agree that testing is an important part of software development. But is testing of research software any different than testing of other types of software? Is testing in a research context different from testing in general software enterprises? And how and what do you test in the first place? What can we learn from each other and from practices elsewhere?
+
+ The call will discuss, and share knowledge and experiences, around questions such as why you would want to test and what the benefits are, what technologies are out there, and what processes RSEs do (and don’t) employ to test their software. No matter if you have never heard of testing or if you are an expert, come join us for an interesting discussion and to hear and share some "testimonies" of what has worked and not worked for others.
```

Comment:
testimonies! :laughing: (sorry still reading rendered version, love this, extra cookies to whomever wrote this!

Reply:
I can't take any credit. I'm guessing Chris or Julia.